apps/meteor/server/methods/sendWelcomeEmail.ts

```diff
@@ -0,0 +1,49 @@
+import { Users } from '@rocket.chat/models';
+import { Meteor } from 'meteor/meteor';
+
+import * as Mailer from '../../app/mailer/server/api';
+import { settings } from '../../app/settings/server';
+import { isSMTPConfigured } from '../../app/utils/server/functions/isSMTPConfigured';
+
+export async function sendWelcomeEmail(to: string): Promise<void> {
+	if (typeof to !== 'string') {
+		throw new Meteor.Error('error-invalid-arguments', 'Invalid arguments', {
+			method: 'sendWelcomeEmail',
+		});
+	}
```

Comment:
Not that I'm against validating, but if the endpoint is already validating this param is an string, is this extra check needed?

Reply:
Because this Fn can be used elsewhere